django/santropolFeast/meal/models.py

```diff
@@ -67,3 +67,25 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class Menu(models.Model):
+
+    class Meta:
+        verbose_name_plural = _('menus')
+
+    # Menu information
+    name = models.CharField(
+        max_length=50,
+        verbose_name=_('name')
+    )
+
+    date = models.DateField(verbose_name=_('date'))
+
+    ingredients = models.ManyToManyField(
+        'menu.Ingredient',
+        related_name='related_menus'
```

Comment:
I'm ok with you. But some related_name respect this pattern.
I propose to open a new issue for this.